<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>ArkSEModpackCreatorAndInstaller</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="label">
            <property name="font">
              <font>
                <pointsize>12</pointsize>
              </font>
            </property>
            <property name="text">
              <string>Path to Ark Survival Evolved:</string>
            </property>
          </widget>
        </item>
        <item row="0" column="1">
          <widget class="QLineEdit" name="gamePathQuery"/>
        </item>
        <item row="0" column="2">
          <widget class="QPushButton" name="browseButton">
            <property name="text">
              <string>...</string>
            </property>
          </widget>
        </item>
        <item row="1" column="0">
          <widget class="QPushButton" name="pushButton">
            <property name="font">
              <font>
                <pointsize>24</pointsize>
              </font>
            </property>
            <property name="text">
              <string>Click me</string>
            </property>
          </widget>
        </item>
        <item row="2" column="0">
          <widget class="QLabel" name="label_2">
            <property name="text">
              <string>Mods:</string>
            </property>
          </widget>
        </item>
        <item row="2" column="1">
          <widget class="QLineEdit" name="modsSteamIdListQuery"/>
        </item>
        <item row="3" column="0">
          <widget class="QPushButton" name="installButton">
            <property name="text">
              <string>Installer mods</string>
            </property>
          </widget>
        </item>
        <item row="4" column="0">
          <widget class="QCheckBox" name="deleteModsCheckBox">
            <property name="text">
              <string>Supprimer le dossier "Mods" avant de copier les mods</string>
            </property>
          </widget>
        </item>
        <item row="4" column="1">
          <widget class="QCheckBox" name="backupModsCheckBox">
            <property name="text">
              <string>Sauvegarder les mods avant d'installer les Mods</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>21</height>
        </rect>
      </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
 </widget>
  <resources/>
  <connections/>
</ui>